runtime/caches/fileSystem.ts

```diff
@@ -26,6 +31,41 @@ const bufferSizeSumObserver = meter.createUpDownCounter("buffer_size_sum", {
   valueType: ValueType.INT,
 });
 
+function numToUint8Array(num: number) {
+  const arr = new Uint8Array(8);
+  for (let i = 0; i < 8; i++) {
+    arr[i] = num % 256;
+    num = Math.floor(num / 256);
+  }
+  return arr;
+}
+
+function uint8ArrayToNum(arr: Uint8Array) {
+  let num = 0;
+  for (let i = 0; i < 8; i++) {
+    num += Math.pow(256, i) * arr[i];
+  }
+  return num;
+}
+
+const cacheOptions = {
+  maxSize: MAX_CACHE_SIZE,
+  ttlAutopurge: TTL_AUTOPURGE,
+  ttlResolution: TTL_RESOLUTION,
+  // deno-lint-ignore no-unused-vars
+  sizeCalculation: (value: Uint8Array, key: string) => {
+    return uint8ArrayToNum(value); // return the length of the array
+  },
+  // deno-lint-ignore no-unused-vars
+  dispose: (value: Uint8Array, key: string) => {
```

Comment:
Me parece que esse dispose por default já remove a chave do LRU, certo?
Pq se remover só o arquivo e a chave ainda existir no cache vai dar uns not found no cache quando fizer o GET/MATCH